services/model.js

```diff
@@ -520,7 +540,7 @@
                 that.set(request.length > 1 ? request[0] : attribute, target)
               }
               if (angular.isUndefined(item)) {
-                that.set(attribute, !target)
+                that.set(attribute, target)
```

Comment:
The toggle feature is supposed to flip the boolean value, so removing this would cause it to do nothing.

Reply:
I removed it already. But I detected a potential problem, in case the `target` is an array and the `item` is undefined the code will set the `target` to be `false`. It means we'll remove all of the items in the array while we are trying to add an `item` into an array.

Now, My code works without modifying this line but I think it should be consideration.

Reply:
This is my test code: https://jsfiddle.net/pdkpro/7y20p6rk/5/

Reply:
An `array` should be toggled to `null`, but I can see the issue you've brought up.  I have a fix rolled out in the commit https://github.com/Sitetheory/stratus/commit/28797e0bcabe9c41394831f93381d24df7607739.